spark/src/main/scala/org/apache/spark/sql/delta/expressions/HilbertIndex.scala

```diff
@@ -0,0 +1,405 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
```

Comment:
nit: copyright years may need updating

Reply:
It looks like every file under `spark/` always uses 2021.